packages/rke2-multus/charts/values.yaml

```diff
@@ -139,3 +152,6 @@ global:
 rke2-whereabouts:
   enabled: false
   
+# When using the thick plugin, manifests.configMap should be set to true
```

Comment:
Should or must? It looks like this PR changes the default to on, how does this affect users on upgrade?

Reply:
should we change the wording here to note that it must be true when using thick?